Lib/test/test_cmd_line.py

```diff
@@ -1054,6 +1054,20 @@ def test_cpu_count_default(self):
         res = assert_python_ok('-c', code, PYTHON_CPU_COUNT='default')
         self.assertEqual(self.res2int(res), (os.cpu_count(), os.process_cpu_count()))
 
+    def test_import_time(self):
+        code = "import os"
+        res = assert_python_ok('-X', 'importtime', '-c', code)
+        self.assertRegex(
+            res.err.decode("utf-8"),
+            r"import time: \s*\d+ | \s*\d+ | \s*os"
```

Comment:
It would be nice to check that this line occurs only once.

You can also check that the line with "cached" is not output.

Reply:
I've added a check to ensure `-Ximporttime` does not include `=2` output. I added a rudimentary check that `-Ximporttime only shows `os` once, though this may be flaky if the structure of the `os` module changes.